src/tools/ips.rs

```diff
@@ -8,4 +8,31 @@
 /// for MacOS:
 /// sudo python3 ./tools/ips.py --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev lo0
 /// For more information read the documentation of the ips.py script.
-pub const IPS: &[&str] = &[];
+use std::fs;
+
+use serde::Deserialize;
+
+const IPS_LIST_PATH: &str = "./tools/ips_list.json";
+
+#[derive(Default, Clone, Deserialize, Debug)]
+struct IpsList {
+    pub nodes: Vec<String>,
+}
+
+/// Read in the node array generated by the ips.py
+/// script.  If file does not exist, return an empty
+/// array.
+fn load_ips_nodes(filepath: &str) -> Vec<String> {
+    let result = fs::read_to_string(filepath);
+    if result.is_err() {
+        println!("no file {filepath}");
+        return Vec::<String>::new();
+    }
+    let jstring = result.unwrap();
+    let ips_list: IpsList = serde_json::from_str(&jstring).unwrap();
+    ips_list.nodes
+}
+
+pub fn ips() -> Vec<String> {
```

Comment:
Let's leave a short fn comment with `///` since it's a public function

Reply:
done